src/food.cpp

```diff
@@ -1,12 +1,46 @@
+#include <iostream>
+
 #include "food.hpp"
 
 
-Food::Food(const sf::Vector2f position) : _shape(size)
+Food::Food() : _type(FoodType::Neutral), _shape(sf::Vector2f(20.f, 20.f))
+{
+    _shape.setPosition(sf::Vector2f{});
+    _shape.setFillColor(sf::Color::White); 
+}
+
+Food::Food(const sf::Vector2f position, const sf::Vector2f size) : _type(FoodType::Neutral), _shape(size)
+{
+    _shape.setPosition(position);
+    _shape.setFillColor(sf::Color::White);
+}
+
+Food::Food(const sf::Vector2f position, const sf::Vector2f size, const FoodType type) :
+    _type(FoodType::Neutral), _shape(size)
 {
     _shape.setPosition(position);
-    _shape.setFillColor(_color);
+    _shape.setFillColor(sf::Color::White);
 }
 
 void Food::draw(sf::RenderTarget & target, sf::RenderStates states) const {
     target.draw(_shape, states);
-}
\ No newline at end of file
+}
+
+void Food::setType(const FoodType type)
+{
+    _type = type;
+    switch (_type)
+    {
+        case FoodType::Neutral:
+            _shape.setFillColor(sf::Color::White);
+            break;
+        case FoodType::Aggressive:
+            _shape.setFillColor(sf::Color::Red);
+            break;
+        case FoodType::Valuable:
+            _shape.setFillColor(sf::Color::Magenta);
+            break;
+        default:
+            std::cout << "Unknown food type!!" << std::endl;
+    }
+}
```

Comment:
Do you need to change the type of the food after creating it? If no, then you don't need this function, move it to the constructor.

Reply:
Now it is need, because i create only one food object.